### ISSUE TITLE
GCODE Preview Legend [Request]

### ISSUE THREAD

**Issue by OverWorldD** [OctoPrint/OctoPrint#4345]
### Is your feature request related to a problem? Please describe.

This feature would only help to clarify the different symbols in the gcode preview.

### Describe the solution you'd like

A Legend under the gcode preview window that explains what the different colors and symbols mean.

**Comment by raymadigan** [comment]
I agree that this request would be useful.  The fact that you use red typically means a problem and the user needs to search for what the red means.  If you don't want to do it, maybe change red to another color?
